west.yml

```diff
@@ -193,7 +193,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 3c4f74b405ca174fe3eb5f3b74c355376ee9a69f
+      revision: db6806f8a8207dd2e219aedd9ed9174f2cde8169
```

Comment:
HAL NXP update needs to be prior to the changes to enable the CSI driver on the RT11xx to preserve bisectability

Reply:
yes - I think you mean MIPI CSI-2 driver ? as hal_update is for MIPI CSI-2

Reply:
Yes please, thanks!